api/v1alpha1/constants.go

```diff
@@ -3,14 +3,32 @@ package v1alpha1
 const (
 	ResourceHashAnnotation = "k8ssandra.io/resource-hash"
 
-	// StargateLabel is the distinctive label for objects created by the Stargate controller.
-	StargateLabel = "k8ssandra.io/stargate"
+	NameLabel      = "app.kubernetes.io/name"
+	NameLabelValue = "k8ssandra-operator"
 
-	DefaultStargateVersion = "1.0.30"
+	InstanceLabel = "app.kubernetes.io/instance"
```

Comment:
These 2 labels are declared but not used for now. I'm not sure how to compute a sensible value for them, and this goes beyond the scope of this PR>

Reply:
Yeah, definitely agree it is outside the scope of the PR. With that said, we do need to firm up our labels.